docs/Homebrew-on-Linux.md

```diff
@@ -47,9 +47,9 @@ If you're using an older distribution of Linux, installing your first package wi
 
 ## Linux/WSL Requirements
 
-+ **GCC** 4.4 or newer
++ **GCC** 4.7.2 or newer
```

Comment:
```suggestion
+ **GCC** 4.7.0 or newer
```

Reply:
I did not test 4.7.0. Not sure which glibc/gcc combination can be found out there. Old distributions have probably updated to the latest minor version of gcc, so I hope most of the people will have 4.7.2. But I may be wrong.

Reply:
The minimum GCC version required to build glibc 2.23 is 4.7.0. That's the reason for this minimum version.

Reply:
See the preamble at https://github.com/Linuxbrew/brew/wiki/Bottles